src/main/java/javatro/core/Round.java

```
@@ -326,4 +451,18 @@ public List<Card> getPlayedCards() {
     public PokerHand getPlayedHand() throws JavatroException {
         return HandResult.evaluateHand(playedCards);
     }
+
+    /**
+     * @warn This method will be deprecated in future versions. Reordering of player hands should be
+     *     done in UI and not in Round class. Updates the player hand.
+     * @param playerHandCards The new holding hand to set
+     * @throws IllegalArgumentException if the player hand is null
+     */
+    public void setPlayerHandCards(List<Card> playerHandCards) {
+        if (playerHandCards == null) {
+            throw new IllegalArgumentException("playerHandCards cannot be null");
```

Exception is not handled here

---

Do you mean JavatroException? This function will only be used for sorting by suit/rank under the CardSelectScreen.

---

Oh, I meant the IllegalArguementException, you throw it here and its not handled, you could either handle it within this function or get the function to throw it (public void setPlayerHandCards(List<Card> playerHandCards throws IllegalArguementException)

---

Also you can just make it a JavatroException